private-templates-service/client/src/components/Common/Tags/Tags.tsx

```diff
@@ -139,7 +125,7 @@ class Tags extends React.Component<Props, State>  {
           <Tag ref={(ref: HTMLDivElement) => this.tagRefs[tag] = ref} onAnimationEnd={this.onAnimationEnd} key={tag}>
             <TagText>{tag}</TagText>
 	    {allowEdit &&
-            <TagCloseIcon key={tag} iconName="ChromeClose" onClick={() => this.tagRemove(tag)} />}
+            <TagCloseIcon key={tag} iconName="ChromeClose" onClick={() => this.props.tagRemove ? this.props.tagRemove(tag) : {} } />}
```

Comment:
sorry, its not exactly clear here what this is doin.. im super confused

I cannot tell if it is evaluating `() => this.props.tagRemove` which is probably going to evaluate to true every time.. evaluating a variable that is a function is probably true, i dont think it will actually run the function unless you do `(() => this.props.tagRemove)()`

maybe it should be `{this.props.tagRemove? () => this.props.tagRemove(tag) : null}` 
I am not certain, maybe you already researched it and landed on this. if so ignore

Reply:
`{ () => this.props.tagRemove ? this.props.tagRemove(tag) : {} }`

It is evaluating this: `(() => this.props.tagRemove) === true` and if that is true, it does the 1st one, otherwise it does the 2nd one. 

I am suggesting you mean to evaluate `this.props.tagRemove === true`

Reply:
`this.props.tagRemove && () => this.props.tagRemove(tag)`